webserver/webserver/src/main/java/io/helidon/webserver/BareResponseImpl.java

```diff
@@ -444,6 +446,9 @@ private ChannelFuture sendData(DataChunk data) {
                         }
                     });
                     data.release();
+                    if (data.flush()) {
```

Comment:
I think the order of `data.release` and `data.flush` may be important?

I didn't look into the ownership of `data`, but typically you are not allowed to access pooled objects after you declared you don't need them (after `data.release()`).

Reply:
Its a final flag set during construction of DataChunk

Reply:
fixed